.travis.yml

```diff
@@ -16,6 +16,8 @@ matrix:
     # Test against dev versions of dependencies
     - php: 7.1
       env: DEPENDENCIES='dev'
+    - php: 7.2
+      env: DEPENDENCIES='dev'
```

Comment:
no need to have 2 different jobs using dev versions of our dependencies. This increases the test matrix for no benefit.